### PR TITLE
Create routes

### PR DESCRIPTION
**Pull request by keento0809**
### Feature Description

1. Created routes for login, signup, favlist view pages
2. Added basic styles to each view pages by referring to page.tsx in `/src/app/map/page.tsx`

### Related Issue

N/A

### Something you want to mention

* As you guys may recognize, I only created routings so it is necessary to add stylings to each view pages later.

### Checklist before requesting a review

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/app/favlist/page.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const FavList = () => {
```
Comment (Lada496) [inline src/app/favlist/page.tsx:7]:
```suggestion
import { type Metadata } from "next";

export const metadata: Metadata = {
  title: "Favorite"
};

const FavList = () => {
```

File: src/app/login/page.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const Login = () => {
```
Comment (Lada496) [inline src/app/login/page.tsx:7]:
```suggestion
import { type Metadata } from "next";

export const metadata: Metadata = {
  title: "Login"
};

const Login = () => {
```

File: src/app/signup/page.tsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const Signup = () => {
```
Comment (Lada496) [inline src/app/signup/page.tsx:7]:
```suggestion
import { type Metadata } from "next";

export const metadata: Metadata = {
  title: "Signup"
};

const Signup = () => {
```

**Review by Lada496 — COMMENTED** [review]
Hi @keento0809 thank you for setting up all routes!
I think we can add matadata a kind of equivalent page router `<Head />`.

I actually set [template](https://nextjs.org/docs/app/api-reference/functions/generate-metadata#template) at root layout before, so just adding title should work.
(Maybe we can talk about better title and description later but we can leave them for now)

ref: https://nextjs.org/docs/app/building-your-application/optimizing/metadata

**Comment by keento0809** [comment]
> Hi @keento0809 thank you for setting up all routes! I think we can add matadata a kind of equivalent page router `<Head />`.
> 
> I actually set [template](https://nextjs.org/docs/app/api-reference/functions/generate-metadata#template) at root layout before, so just adding title should work. (Maybe we can talk about better title and description later but we can leave them for now)
> 
> ref: https://nextjs.org/docs/app/building-your-application/optimizing/metadata

Oh yeah that's true, I should have added metadata to each route.
I'll add them based on your suggested codes!

**Comment by Lada496** [comment]
@keento0809 thank you for the updates!

**Review by devjoylee — APPROVED** [review]
LGTM!

**Review by Lada496 — APPROVED** [review]

**Pull request merged**